books/free-programming-books-cs.md

```diff
@@ -1,26 +1,26 @@
 ### Index
 
+* [Algoritmy a datové struktury](#algoritmy-a-datove-struktury)
 * [Bash](#bash)
+* [Bezpečnost](#bezpecnost)
 * [C#](#csharp)
 * [C++](#cpp)
+* [Distribuce](#distribuce)
+* [Django](#django)
 * [Git](#git)
 * [HTML and CSS](#html-and-css)
 * [Java](#java)
 * [Language Agnostic](#language-agnostic)
-    * [Algoritmy a datové struktury](#algoritmy-a-datove-struktury)
```

Comment:
Why are all these lines removed?

Reply:
i didn't removed them but rather changed their position in the file as well as i removed extra whitespaces which were causing errors so as to make it "alphabatical"

Reply:
Please leave them as is. They are sub-sections. so that should not be the problem

Reply:
done sir
now it is showing All checks have passed.
thank you sir!!